### PR TITLE
Add simple script to create release archives

### PR DESCRIPTION
**Pull request by davidscn**
...which might come in handy in the future.

### CODE REVIEW

File: tools/create-release-archives.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```
Comment (MakisH) [inline tools/create-release-archives.sh:2]:
```suggestion
# Execute this script from the top-level directory (./tools/create-release-archives.sh)

```

File: tools/create-release-archives.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+remote_name=upstream
+adapter_version="v1.2.3"
+
+# Declare branch names for which archives will be created
+declare -a branches=("master" "OpenFOAMv1806" "OpenFOAM10" "OpenFOAM9" "OpenFOAM8" "OpenFOAM7" "OpenFOAM6" "OpenFOAM5" "OpenFOAM4")
+
+for i in "${branches[@]}"
+    do
+    archive_name=openefoam-adapter-"${adapter_version}"-"${i}"
+    git archive --format=tar.gz --prefix="${archive_name}"/ "${remote_name}"/"${i}" >"${archive_name}".tar.gz
```
Comment (MakisH) [inline tools/create-release-archives.sh:14]: Having the archives created in a separate directory would make overview and cleanup easier:

```suggestion
    mkdir -p release-archives
    git archive --format=tar.gz --prefix="${archive_name}"/ "${remote_name}"/"${i}" >"release-archives/${archive_name}".tar.gz
```

File: tools/create-release-archives.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -u
+
+remote_name=upstream
+adapter_version="v1.2.3"
+
+# Declare branch names for which archives will be created
+declare -a branches=("master" "OpenFOAMv1806" "OpenFOAM10" "OpenFOAM9" "OpenFOAM8" "OpenFOAM7" "OpenFOAM6" "OpenFOAM5" "OpenFOAM4")
+
```
Comment (davidscn) [inline tools/create-release-archives.sh:10]:
```suggestion

mkdir -p release-archives
```

File: tools/create-release-archives.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Execute this script from the top-level directory (./tools/create-release-archives.sh)
+
+set -u
+
+remote_name=origin
+adapter_version="v1.2.3"
```
Comment (davidscn) [inline tools/create-release-archives.sh:7]: I could imagine that we could even get both of these information from git, but let's don't overdo it right now..

**Review by MakisH — APPROVED** [review]

**Pull request merged**